src/main/java/frc/trigon/robot/utilities/Conversions.java

```diff
@@ -9,7 +9,11 @@ public static double hundredMsToMin(double ms) {
         return hundredMsToSec(ms) * 60;
     }
 
-    public static double hundredMsToTickPerMin(double ms) {
-        return hundredMsToMin(ms) * 2048;
+    public static double ticksToRevolutions(double ticks) {
+        return ticks / 4048;
+    }
+
+    public static double ticksPerHundredMsToRPM(double velocityInHundredMs) {
+        return hundredMsToMin(velocityInHundredMs) / ticksToRevolutions(velocityInHundredMs);
     }
```

Comment:
This doesn't work. you can't separate the velocity into revolutions and time and then put it back together. you need to just times the given velocity by the conversion to minutes and divide by the conversion from ticks to revolutions. you can leave the other functions even though you don't need them because we may use them in other places.

Reply:
You can set constants at the top of the class with the conversions so there aren't just magic numbers in the middle of functions.